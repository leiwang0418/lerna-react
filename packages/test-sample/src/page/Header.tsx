import React from 'react';

const Header = () => (<div>请输入要查询的github账号:</div>);

export default Header;