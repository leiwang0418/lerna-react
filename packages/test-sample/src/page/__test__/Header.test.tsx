import React from 'react';
import { render, screen } from '@testing-library/react';

import Header from '../Header';

describe('Header render test', () => {
	test('renders the basic header', () => {
		const { container } = render(<Header />);
		const headerElement = screen.getByText(/请输入要查询的github账号:/i);
		expect(headerElement).toBeInTheDocument();
		
		expect(container).toMatchSnapshot();
	});
});
