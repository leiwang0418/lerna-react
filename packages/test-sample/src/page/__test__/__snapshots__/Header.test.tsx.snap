// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render test renders the basic header 1`] = `
<div>
  <div>
    请输入要查询的github账号:
  </div>
</div>
`;
